src/app/admin/demands/demands-list/demands-list.component.ts

```diff
@@ -210,22 +209,10 @@ export class DemandsListComponent implements OnInit {
 
   onExport() {
     this.demandsFacade.getExportDemands().subscribe((res) => {
-      this.downloadCsv(res);
+      ExportCSVService.uploadCsv(res, 'demands');
```

Comment:
bellow I said to rename `downloadCsv` to be a normal function not `static` so here
Use dependency injection in the right way, add `ExportCSVService` in constructor `private exportCsvService: ExportCSVService`

and here use `exportCsvService.downloadCsv(res, 'demands');`